index.bs

```diff
@@ -2335,7 +2335,7 @@ follows.
 
 ### <dfn>Signature Counter</dfn> Considerations ### {#sign-counter}
 
-Authenticators MUST implement a [=signature counter=] feature. The [=signature counter=] is incremented for each successful
+Authenticators SHOULD implement a [=signature counter=] feature. The [=signature counter=] is incremented for each successful
```

Comment:
Changing this to SHOULD as authenticators which are purely server based (non-resident), can't store anything. We support such kind of authenticators in FIDO2.